templates/devspace.yaml.tpl

```diff
@@ -307,6 +283,10 @@ hooks:
   - name: auth-refresh
     command: "${DEVENV_BIN} --skip-update auth refresh"
     events: ["before:build"]
+  # Since we do not use LOFT put the KIND image loading here
```

Comment:
Does this hook in only when told to build a local image, we should only need it then